File: semester_1/python_programming/CYCLE 3/6.py
count = 0
n = int(input("Enter the number : "))
for j in range(2, n + 1):
    flag = 0
    for i in range(2, j):
        if j % i == 0:
            flag = 1
    if flag != 1:
        if count % 2 == 0:
            print(j, end = " ")
        count = count + 1
    

File: semester_1/web_technology/PHP PRACTICE/6.php
<html>
<head>
    <title>SUM</title>
</head>
<body>
    <h2>Add Two Numbers</h2>

    <form method="post">
        <label for="number1">Enter the first number:</label>
        <input type="number" name="number1" required>
        <br><br>
        <label for="number2">Enter the second number:</label>
        <input type="number" name="number2" required>
        <br><br>
        <button type="submit">Add Numbers</button>
    </form>

    <?php
        if ($_SERVER["REQUEST_METHOD"] == "POST") {
            $number1 = $_POST["number1"];
            $number2 = $_POST["number2"];
            
            $sum = $number1 + $number2;
            echo "$sum";
        }
    ?>
</body>
</html>